toolkit/docs/building/prerequisites-mariner.sh

```diff
@@ -15,7 +56,7 @@ tdnf -y install \
     genisoimage \
     git \
     glibc-devel \
-    golang \
+    golang-1.24.3 \
```

Comment:
This will need to handle 2.0 as well, which needs `msft-golang`. Many build machines will likely still be on 2.0.